docs/reference/deploy/config/agent-config-reference.rst

```diff
@@ -198,3 +198,11 @@ Deprecated. This field has been deprecated and will be ignored. Use ``resource_p
 
 If ``true``, enables a more verbose form of logging that may be helpful in diagnosing issues.
 Defaults to ``false``.
+
+****************
+ ``image_root``
+****************
+
+If set then specifies the path to a shared directory of previously downloaded Determined environment
```

Comment:
Probably could have gotten away with "Specifies the path ...", without the "If set then".

Reply:
Actually, probably "The path" would be sufficient too.